src/app/public/modules/summary-actionbar/summary-actionbar.component.html

```diff
@@ -0,0 +1,58 @@
+<div
+  class="sky-summary-actionbar"
+  [ngClass]="{'sky-summary-actionbar-summary-collapsable': summaryCollapseMode, 'sky-summary-actionbar-modal': inModalFooter}"
```

Comment:
Can you put these classes on multiple lines?

Reply:
Done